packages/central-server/app/sync/getLookupSourceTickRange.js

```diff
@@ -0,0 +1,33 @@
+import { Op } from 'sequelize';
+
+// After the regular sync snapshotting process is complete, we need to grab any changelog records
+// that have an updated_at_sync_tick matching the range of lookup table source ticks being synced.
+// This avoids having to run any complex logic for filtering the correct changelog records to
+// send to a facility, as the snapshot has already run the right logic across the raw records, so
+// if a record is included there, we know the recent changelog entries should be included too.
+
+export const getLookupSourceTickRange = async (
+  { models },
+  pullSince,
+  pullUntil,
+) => {
+  const { SyncLookupTick } = models;
+  const lookupTicks = await SyncLookupTick.findAll({
+    where: {
+      lookupEndTick: {
+        [Op.gt]: pullSince,
```

Comment:
@passcod potential place here, alternatives include
- sourceStartTick > pullSince
- lookupEndTick >= pullSince
- sourceStartTick >= pullSince
Just listing out all the possibilities so I can think them through

Reply:
@dannash100 the most generous of these would be `sourceStartTick >= pullSince`, chuck that in your testing and see if things get better. If not, we know pretty confidently the issue is not in this function.